kolibri_explore_plugin/views.py

```diff
@@ -39,12 +44,10 @@ def options(self, request, *args, **kwargs):
         return HttpResponse()
 
     def _get_file(self, app, path):
-        base = os.path.join(os.path.dirname(__file__), "apps")
-
         if path.startswith("/"):
             path = path[1:]
 
-        filename = os.path.join(base, app, path)
+        filename = os.path.join(APPS_BUNDLE_PATH, app, path)
```

Comment:
What do you think about making this a `PATH` resolution, with multiple paths with priority, instead of just a fixed path? For example:

```python
filename = None
for apps_path in APPS_PATHS:
    filename = os.path.join(apps_path, app, path)
    if os.path.exists(filename):
        break
else:
    raise Http404
```

And `APPS_PATHS` could be a composition of the option and the fallback path inside the python code.

Reply:
Thanks! That makes sense to me. It flows nicer :) It would help us with including a _default_ app in the explore plugin itself. This way we would have a reasonable guarantee that it exists.